ci-operator/step-registry/azure/provision/bastionhost/azure-provision-bastionhost-commands.sh

```diff
@@ -117,8 +138,8 @@ echo "Create a Storage Account for bastion vhd"
 # 'account_name' must have length less than 24, so hardcode the basion sa name
 sa_name_prefix=$(echo "${NAMESPACE}" | sed "s/ci-op-//" | sed 's/[-_]//g')
 sa_name="${sa_name_prefix}${UNIQUE_HASH}basa"
-run_command "az storage account create -g ${bastion_rg} --name ${sa_name} --kind Storage --sku Standard_LRS" &&
-account_key=$(az storage account keys list -g ${bastion_rg} --account-name ${sa_name} --query "[0].value" -o tsv) || exit 3
+run_command_with_retries "az storage account create -g ${bastion_rg} --name ${sa_name} --kind Storage --sku Standard_LRS"
```

Comment:
the default max retries of `run_command_with_retries` is 20, is that a bit agressive?

Reply:
changed to retry 5 times for this command, the function is verified with default max retires in [job](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/46954/rehearse-46954-periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-stable-azure-ipi-fullyprivate-proxy-f14/1737374560879120384/artifacts/azure-ipi-fullyprivate-proxy-f14/azure-provision-bastionhost/build-log.txt) and passed.